flink-runtime/src/main/java/org/apache/flink/runtime/leaderretrieval/ZooKeeperLeaderRetrievalService.java

```diff
@@ -191,8 +191,6 @@ protected void handleStateChange(ConnectionState newState) {
 				LOG.debug("Connected to ZooKeeper quorum. Leader retrieval can start.");
 				break;
 			case SUSPENDED:
-				LOG.warn("Connection to ZooKeeper suspended. Can no longer retrieve the leader from " +
```

Comment:
The same as https://github.com/apache/flink/pull/9158/files#r304770340

Reply:
Thanks, done.